bin/compile-requirements.sh

```diff
@@ -10,21 +10,11 @@ export CUSTOM_COMPILE_COMMAND="make compile-requirements"
 
 # We need this installed, but we don't want it to live in the main requirements
 # We will need to periodically review this pinning
-pip install --upgrade pip-tools==6.4.0  # needs at least this version to build
+pip install pip-tools==6.4.0  # needs at least this version to build
```

Comment:
If it's at least this version, can we unpin? Or is there something in a higher version that is no good?

Reply:
I think pinning is important for consistent builds. We don't want a new release to block us until we figure it out. I know this doesn't make it to prod or anything, but it's still a bit of our infra. I even get nervous when I see a pip call with no hashes :) It's a tiny security risk, but it is one.

Reply:
Rob spotted that pip-tools and pip are back in sync so I have bumped the latest versions to suit - see 2d79fec36